docs/docs/deploying-to-gitlab-pages.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Deploying to GitLab Pages
+---
+
+GitLab Pages are similar to GitHub Pages, perhaps even easier to setup. It also
+supports custom domain names and SSL certificates. The process of setting GitLab
+pages up is made a lot easier with GitLab's included continuous integration
+platform.
+
+Create a new GitLab repository, initialize your Gatsby project folder if you
```

Comment:
What do you mean by "initialize your Gatsby project folder"? Would it make sense to link to [quick start](/docs/)?